README.md

```diff
@@ -56,4 +71,4 @@ This repository integrates many other repos as subdirectories using the [git sub
 * `git subtree pull --squash --prefix lld https://llvm.org/git/lld.git master`
 * `git subtree pull --squash --prefix llvm https://llvm.org/git/llvm.git master`
 * `git subtree pull --squash --prefix musl https://github.com/jfbastien/musl wasm-prototype-1`
-* `git subtree pull --squash --prefix WAVM https://github.com/WAVM/WAVM master`
\ No newline at end of file
+* `git subtree pull --squash --prefix WAVM https://github.com/WAVM/WAVM master`

```

Comment:
Don't know why this is showing in the diff. Perhaps EOF

Reply:
Probably the difference between `<LF><EOF>` and `<EOF>`.